gateway/xgress_edge/listener.go

```diff
@@ -158,7 +158,7 @@ func (proxy *ingressProxy) processConnect(req *channel2.Message, ch channel2.Cha
 		return
 	}
 
```

Comment:
I feel like we assume this is the token in some other places. Changing this may require a migration and updating some other bits of code. 

I'm not 100% sure of that, though. If you haven't vetted it yet, though, that would be something to look into

Reply:
What this affects is the `clientId` field in the fabric controller's in-memory session table that isn't persisted.

From the fabric's perspective, I saw it used to publish the fabric session create/delete events, but that was it.